composer.json

```diff
@@ -11,7 +11,7 @@
   ],
   "require": {
     "php": ">=5.3.3",
-    "ezsystems/ezpublish-kernel": "~5.3 | >=2014.03"
+    "ezsystems/ezpublish-kernel": ">5.3 | >6.0 | >7.0"
```

Comment:
Nitpicks:
1. '||' is used in composer docs for OR: https://getcomposer.org/doc/articles/versions.md#version-range
2. '>5.3' does include 6, 7, 8 and up... what about using '^5.3 || ^6.0 || ^7.0'
3. I'd leave in '>=2014.03' for those still using ezpublish-platform community edition 5.4